modules/microshift-custom-ca-cert-rotation.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-cert-rotation_{context}"]
+= Configuring `oc` to start with an expired custom certificates
+
+External server certificates are not automatically renewed. You must manually rotate your external certificates. However, you can configure the {oc-first} command-line tool to allow a start with an expired certificate. Then you can rotate the certificates as needed.
+
+.Procedure
+
+. To allow a system start with an expired custom certificate, use the `--insecure-skip-tls-verify` `oc` mode by adding the following example configuration file to the host:
```

Comment:
I do not think it's create what `system start` means and what configuration file you refer to. Do you mean one needs to edit kubeconfig file?